gdx/build.gradle

```diff
@@ -29,6 +29,7 @@ compileJava {
 }
 
 dependencies {
+    compileOnly "org.joml:joml:1.9.8"
```

Comment:
Left in by mistake?

Reply:
Sorry, yes. I tried to experiment with JOML. I'll delete the dependency.